Subject: [PATCH] 1.20.4 build
---
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision a0b13e4622ca4435f7c4a5d5a796228a44b17435)
+++ b/gradle.properties	(date 1731184050775)
@@ -6,8 +6,8 @@
 
 # Fabric Properties
 # check these on https://fabricmc.net/develop
-minecraft_version=1.21
-yarn_mappings=1.21+build.2
+minecraft_version=1.20.4
+yarn_mappings=1.20.4+build.3
 loader_version=0.15.11
 
 mod_version=0.5.1
@@ -16,8 +16,8 @@
 
 
 # Dependencies
-fabric_version=0.100.3+1.21
+fabric_version=0.97.2+1.20.4
 
-modmenu_version=11.0.1
-yacl_version=3.5.0+1.21-fabric
+modmenu_version=9.0.0
+yacl_version=3.5.0+1.20.4-fabric
 
Index: src/main/java/circuitlord/reactivemusic/SongPicker.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/circuitlord/reactivemusic/SongPicker.java b/src/main/java/circuitlord/reactivemusic/SongPicker.java
--- a/src/main/java/circuitlord/reactivemusic/SongPicker.java	(revision a0b13e4622ca4435f7c4a5d5a796228a44b17435)
+++ b/src/main/java/circuitlord/reactivemusic/SongPicker.java	(date 1731183907984)
@@ -2,7 +2,10 @@
 
 
 import circuitlord.reactivemusic.mixin.BossBarHudAccessor;
-import net.fabricmc.fabric.api.tag.convention.v2.ConventionalBiomeTags;
+//import net.fabricmc.fabric.api.tag.convention.v2.ConventionalBiomeTags;
+import net.fabricmc.fabric.api.tag.convention.v1.ConventionalBiomeTags;
+
+
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.client.gui.hud.BossBarHud;
 import net.minecraft.client.gui.hud.ClientBossBar;
Index: src/main/resources/reactivemusic.mixins.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/reactivemusic.mixins.json b/src/main/resources/reactivemusic.mixins.json
--- a/src/main/resources/reactivemusic.mixins.json	(revision a0b13e4622ca4435f7c4a5d5a796228a44b17435)
+++ b/src/main/resources/reactivemusic.mixins.json	(date 1731183907992)
@@ -1,7 +1,7 @@
 {
   "required": true,
   "package": "circuitlord.reactivemusic.mixin",
-  "compatibilityLevel": "JAVA_21",
+  "compatibilityLevel": "JAVA_17",
   "mixins": [
   ],
   "injectors": {
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision a0b13e4622ca4435f7c4a5d5a796228a44b17435)
+++ b/build.gradle	(date 1731183907967)
@@ -78,7 +78,7 @@
 }
 
 tasks.withType(JavaCompile).configureEach {
-	it.options.release = 21
+	it.options.release = 17
 }
 
 java {
@@ -87,8 +87,8 @@
 	// If you remove this line, sources will not be generated.
 	withSourcesJar()
 
-	sourceCompatibility = JavaVersion.VERSION_21
-	targetCompatibility = JavaVersion.VERSION_21
+	sourceCompatibility = JavaVersion.VERSION_17
+	targetCompatibility = JavaVersion.VERSION_17
 }
 
 jar {
Index: src/main/resources/fabric.mod.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/fabric.mod.json b/src/main/resources/fabric.mod.json
--- a/src/main/resources/fabric.mod.json	(revision a0b13e4622ca4435f7c4a5d5a796228a44b17435)
+++ b/src/main/resources/fabric.mod.json	(date 1731184504252)
@@ -27,8 +27,8 @@
 	],
 	"depends": {
 		"fabricloader": ">=0.15.11",
-		"minecraft": ["~1.21"],
-		"java": ">=21",
+		"minecraft": ["~1.20.4"],
+		"java": ">=17",
 		"yet_another_config_lib_v3": "*"
 	},
 	"custom": {
