Subject: [PATCH] build 1.21 - 1.21.1
---
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision adff89eea1d45fcacaedd9b6c5562df8f8f4ce8c)
+++ b/gradle.properties	(date 1735590405278)
@@ -6,8 +6,8 @@
 
 # Fabric Properties
 # check these on https://fabricmc.net/develop
-minecraft_version=1.21.4
-yarn_mappings=1.21.4+build.4
+minecraft_version=1.21
+yarn_mappings=1.21+build.9
 loader_version=0.16.9
 
 mod_version=1.0.0
@@ -16,8 +16,8 @@
 
 
 # Dependencies
-fabric_version=0.113.0+1.21.4
+fabric_version=0.102.0+1.21
 
-modmenu_version=13.0.0-beta.1
-yacl_version=3.6.2+1.21.4-fabric
+modmenu_version=11.0.3
+yacl_version=3.6.2+1.21-fabric
 
Index: src/main/java/circuitlord/reactivemusic/SongPicker.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/circuitlord/reactivemusic/SongPicker.java b/src/main/java/circuitlord/reactivemusic/SongPicker.java
--- a/src/main/java/circuitlord/reactivemusic/SongPicker.java	(revision adff89eea1d45fcacaedd9b6c5562df8f8f4ce8c)
+++ b/src/main/java/circuitlord/reactivemusic/SongPicker.java	(date 1735590438197)
@@ -171,8 +171,8 @@
         songpackEventMap.put(SongpackEventType.UNDERWATER, player.isSubmergedInWater());
 
         // Weather
-        songpackEventMap.put(SongpackEventType.RAIN, world.isRaining() && biome.value().getPrecipitation(pos, world.getSeaLevel()) == Biome.Precipitation.RAIN);
-        songpackEventMap.put(SongpackEventType.SNOW, world.isRaining() && biome.value().getPrecipitation(pos, world.getSeaLevel()) == Biome.Precipitation.SNOW);
+        songpackEventMap.put(SongpackEventType.RAIN, world.isRaining() && biome.value().getPrecipitation(pos) == Biome.Precipitation.RAIN);
+        songpackEventMap.put(SongpackEventType.SNOW, world.isRaining() && biome.value().getPrecipitation(pos) == Biome.Precipitation.SNOW);
 
         songpackEventMap.put(SongpackEventType.STORM, world.isThundering());
 
Index: src/main/resources/fabric.mod.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/fabric.mod.json b/src/main/resources/fabric.mod.json
--- a/src/main/resources/fabric.mod.json	(revision adff89eea1d45fcacaedd9b6c5562df8f8f4ce8c)
+++ b/src/main/resources/fabric.mod.json	(date 1735590532980)
@@ -27,7 +27,7 @@
 	],
 	"depends": {
 		"fabricloader": ">=0.15.11",
-		"minecraft": ["~1.21"],
+		"minecraft": ["1.21", "1.21.1"],
 		"java": ">=21",
 		"yet_another_config_lib_v3": "*"
 	},
