Subject: [PATCH] build ~1.20.1
---
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision ba2aa82be7ac49e6bb69874e73f1e796706c5cf3)
+++ b/gradle.properties	(date 1735591867870)
@@ -6,8 +6,8 @@
 
 # Fabric Properties
 # check these on https://fabricmc.net/develop
-minecraft_version=1.21.4
-yarn_mappings=1.21.4+build.4
+minecraft_version=1.20.1
+yarn_mappings=1.20.1+build.10
 loader_version=0.16.9
 
 mod_version=1.0.0
@@ -16,8 +16,8 @@
 
 
 # Dependencies
-fabric_version=0.113.0+1.21.4
+fabric_version=0.92.3+1.20.1
 
-modmenu_version=13.0.0-beta.1
-yacl_version=3.6.2+1.21.4-fabric
+modmenu_version=7.2.2
+yacl_version=3.6.2+1.20.1-fabric
 
Index: src/main/java/circuitlord/reactivemusic/SongPicker.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/circuitlord/reactivemusic/SongPicker.java b/src/main/java/circuitlord/reactivemusic/SongPicker.java
--- a/src/main/java/circuitlord/reactivemusic/SongPicker.java	(revision ba2aa82be7ac49e6bb69874e73f1e796706c5cf3)
+++ b/src/main/java/circuitlord/reactivemusic/SongPicker.java	(date 1735591497824)
@@ -3,7 +3,7 @@
 
 import circuitlord.reactivemusic.config.ModConfig;
 import circuitlord.reactivemusic.mixin.BossBarHudAccessor;
-import net.fabricmc.fabric.api.tag.convention.v2.ConventionalBiomeTags;
+import net.fabricmc.fabric.api.tag.convention.v1.ConventionalBiomeTags;
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.client.gui.screen.CreditsScreen;
 import net.minecraft.client.network.ClientPlayerEntity;
@@ -174,8 +174,8 @@
         songpackEventMap.put(SongpackEventType.UNDERWATER, player.isSubmergedInWater());
 
         // Weather
-        songpackEventMap.put(SongpackEventType.RAIN, world.isRaining() && biome.value().getPrecipitation(pos, world.getSeaLevel()) == Biome.Precipitation.RAIN);
-        songpackEventMap.put(SongpackEventType.SNOW, world.isRaining() && biome.value().getPrecipitation(pos, world.getSeaLevel()) == Biome.Precipitation.SNOW);
+        songpackEventMap.put(SongpackEventType.RAIN, world.isRaining() && biome.value().getPrecipitation(pos) == Biome.Precipitation.RAIN);
+        songpackEventMap.put(SongpackEventType.SNOW, world.isRaining() && biome.value().getPrecipitation(pos) == Biome.Precipitation.SNOW);
 
         songpackEventMap.put(SongpackEventType.STORM, world.isThundering());
 
Index: src/main/resources/reactivemusic.mixins.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/reactivemusic.mixins.json b/src/main/resources/reactivemusic.mixins.json
--- a/src/main/resources/reactivemusic.mixins.json	(revision ba2aa82be7ac49e6bb69874e73f1e796706c5cf3)
+++ b/src/main/resources/reactivemusic.mixins.json	(date 1735591647537)
@@ -1,7 +1,7 @@
 {
   "required": true,
   "package": "circuitlord.reactivemusic.mixin",
-  "compatibilityLevel": "JAVA_21",
+  "compatibilityLevel": "JAVA_17",
   "mixins": [
   ],
   "injectors": {
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision ba2aa82be7ac49e6bb69874e73f1e796706c5cf3)
+++ b/build.gradle	(date 1735591713350)
@@ -78,7 +78,7 @@
 }
 
 tasks.withType(JavaCompile).configureEach {
-	it.options.release = 21
+	it.options.release = 17
 }
 
 java {
@@ -87,8 +87,8 @@
 	// If you remove this line, sources will not be generated.
 	withSourcesJar()
 
-	sourceCompatibility = JavaVersion.VERSION_21
-	targetCompatibility = JavaVersion.VERSION_21
+	sourceCompatibility = JavaVersion.VERSION_17
+	targetCompatibility = JavaVersion.VERSION_17
 }
 
 jar {
Index: src/main/resources/fabric.mod.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/fabric.mod.json b/src/main/resources/fabric.mod.json
--- a/src/main/resources/fabric.mod.json	(revision ba2aa82be7ac49e6bb69874e73f1e796706c5cf3)
+++ b/src/main/resources/fabric.mod.json	(date 1735591640143)
@@ -27,8 +27,8 @@
 	],
 	"depends": {
 		"fabricloader": ">=0.15.11",
-		"minecraft": ["~1.21"],
-		"java": ">=21",
+		"minecraft": ["~1.20"],
+		"java": ">=17",
 		"yet_another_config_lib_v3": "*"
 	},
 	"custom": {
