Subject: [PATCH] build 1.20.1
---
Index: src/main/resources/fabric.mod.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/fabric.mod.json b/src/main/resources/fabric.mod.json
--- a/src/main/resources/fabric.mod.json	(revision 83462c7e28f234e1b430af328fdc3edd79893d9d)
+++ b/src/main/resources/fabric.mod.json	(date 1753172941636)
@@ -27,8 +27,8 @@
 	],
 	"depends": {
 		"fabricloader": ">=0.15.11",
-		"minecraft": ["1.21", "1.21.1"],
-		"java": ">=21",
+		"minecraft": [">=1.20.1"],
+		"java": ">=17",
 		"yet_another_config_lib_v3": "*"
 	},
 	"custom": {
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision 83462c7e28f234e1b430af328fdc3edd79893d9d)
+++ b/build.gradle	(date 1753172974159)
@@ -78,7 +78,7 @@
 }
 
 tasks.withType(JavaCompile).configureEach {
-	it.options.release = 21
+	it.options.release = 17
 }
 
 java {
@@ -87,8 +87,8 @@
 	// If you remove this line, sources will not be generated.
 	withSourcesJar()
 
-	sourceCompatibility = JavaVersion.VERSION_21
-	targetCompatibility = JavaVersion.VERSION_21
+	sourceCompatibility = JavaVersion.VERSION_17
+	targetCompatibility = JavaVersion.VERSION_17
 }
 
 jar {
Index: src/main/java/circuitlord/reactivemusic/SongPicker.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/circuitlord/reactivemusic/SongPicker.java b/src/main/java/circuitlord/reactivemusic/SongPicker.java
--- a/src/main/java/circuitlord/reactivemusic/SongPicker.java	(revision 83462c7e28f234e1b430af328fdc3edd79893d9d)
+++ b/src/main/java/circuitlord/reactivemusic/SongPicker.java	(date 1753173033757)
@@ -3,7 +3,7 @@
 
 import circuitlord.reactivemusic.config.ModConfig;
 import circuitlord.reactivemusic.mixin.BossBarHudAccessor;
-import net.fabricmc.fabric.api.tag.convention.v2.ConventionalBiomeTags;
+import net.fabricmc.fabric.api.tag.convention.v1.ConventionalBiomeTags;
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.client.gui.screen.CreditsScreen;
 import net.minecraft.client.network.ClientPlayerEntity;
Index: src/main/resources/reactivemusic.mixins.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/reactivemusic.mixins.json b/src/main/resources/reactivemusic.mixins.json
--- a/src/main/resources/reactivemusic.mixins.json	(revision 83462c7e28f234e1b430af328fdc3edd79893d9d)
+++ b/src/main/resources/reactivemusic.mixins.json	(date 1753172955031)
@@ -1,7 +1,7 @@
 {
   "required": true,
   "package": "circuitlord.reactivemusic.mixin",
-  "compatibilityLevel": "JAVA_21",
+  "compatibilityLevel": "JAVA_17",
   "mixins": [
   ],
   "injectors": {
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision 83462c7e28f234e1b430af328fdc3edd79893d9d)
+++ b/gradle.properties	(date 1753172865883)
@@ -7,18 +7,18 @@
 # Fabric Properties
 # check these on https://fabricmc.net/develop
 
-minecraft_version=1.21
-yarn_mappings=1.21+build.9
+minecraft_version=1.20.1
+yarn_mappings=1.20.1+build.10
 loader_version=0.16.14
 loom_version=1.11-SNAPSHOT
 
 # Fabric API
-fabric_version=0.102.0+1.21
+fabric_version=0.92.6+1.20.1
 
 mod_version=1.1.0
 maven_group=circuitlord.reactivemusic
 archives_base_name=reactivemusic
 
-modmenu_version=11.0.3
-yacl_version=3.6.2+1.21-fabric
+modmenu_version=7.2.2
+yacl_version=3.6.6+1.20.1-fabric
 
