Subject: [PATCH] build 1.21.5 patch
---
Index: src/main/resources/fabric.mod.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/fabric.mod.json b/src/main/resources/fabric.mod.json
--- a/src/main/resources/fabric.mod.json	(revision 73a37a8a4155e10af469894cb6fb31c986ec7087)
+++ b/src/main/resources/fabric.mod.json	(date 1753172520072)
@@ -27,7 +27,7 @@
 	],
 	"depends": {
 		"fabricloader": ">=0.15.11",
-		"minecraft": ["1.21", "1.21.1"],
+		"minecraft": [">=1.21.5"],
 		"java": ">=21",
 		"yet_another_config_lib_v3": "*"
 	},
Index: src/main/java/circuitlord/reactivemusic/SongPicker.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/circuitlord/reactivemusic/SongPicker.java b/src/main/java/circuitlord/reactivemusic/SongPicker.java
--- a/src/main/java/circuitlord/reactivemusic/SongPicker.java	(revision 73a37a8a4155e10af469894cb6fb31c986ec7087)
+++ b/src/main/java/circuitlord/reactivemusic/SongPicker.java	(date 1753172421540)
@@ -174,8 +174,8 @@
         songpackEventMap.put(SongpackEventType.UNDERWATER, player.isSubmergedInWater());
 
         // Weather
-        songpackEventMap.put(SongpackEventType.RAIN, world.isRaining() && biome.value().getPrecipitation(pos) == Biome.Precipitation.RAIN);
-        songpackEventMap.put(SongpackEventType.SNOW, world.isRaining() && biome.value().getPrecipitation(pos) == Biome.Precipitation.SNOW);
+        songpackEventMap.put(SongpackEventType.RAIN, world.isRaining() && biome.value().getPrecipitation(pos, world.getSeaLevel()) == Biome.Precipitation.RAIN);
+        songpackEventMap.put(SongpackEventType.SNOW, world.isRaining() && biome.value().getPrecipitation(pos, world.getSeaLevel()) == Biome.Precipitation.SNOW);
 
         songpackEventMap.put(SongpackEventType.STORM, world.isThundering());
 
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision 73a37a8a4155e10af469894cb6fb31c986ec7087)
+++ b/gradle.properties	(date 1753172280274)
@@ -7,18 +7,19 @@
 # Fabric Properties
 # check these on https://fabricmc.net/develop
 
-minecraft_version=1.21
-yarn_mappings=1.21+build.9
+
+minecraft_version=1.21.5
+yarn_mappings=1.21.5+build.1
 loader_version=0.16.14
 loom_version=1.11-SNAPSHOT
 
 # Fabric API
-fabric_version=0.102.0+1.21
+fabric_version=0.128.1+1.21.5
 
 mod_version=1.1.0
 maven_group=circuitlord.reactivemusic
 archives_base_name=reactivemusic
 
-modmenu_version=11.0.3
-yacl_version=3.6.2+1.21-fabric
+modmenu_version=14.0.0-rc.2
+yacl_version=3.7.1+1.21.5-fabric
 
